src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/dq/DirectionsQuestionnaireGenerator.java

```diff
@@ -245,6 +293,38 @@ private boolean isRespondent2(CaseData caseData) {
         var respondent = caseData.getRespondent1();
         var respondentRepresentative = representativeService.getRespondent1Representative(caseData);
 
+        if (isClaimantResponse(caseData) && (YES
+            .equals(caseData.getApplicant1ProceedWithClaimAgainstRespondent1MultiParty1v2())
+            || YES.equals(caseData.getApplicant1ProceedWithClaimAgainstRespondent2MultiParty1v2()))) {
+
+            ArrayList<Party> respondents = new ArrayList<>();
```

Comment:
Standard way to write this is List<Party> respondents = new ArrayList<>();

Reply:
this would cause this error:
![image](https://user-images.githubusercontent.com/12885268/153205385-ced208b0-d637-4bdd-956c-9a382e83324d.png)

Reply:
You have to specify list is of party type List<Party> respondents = new ArrayList<>(); Then you should not see any error.

Reply:
Done